src/EA.Weee.Web/Areas/Producer/Controllers/ProducerController.cs

```diff
@@ -162,6 +167,11 @@ public async Task<ActionResult> Submissions(int? year = null)
             submissionService.WithSubmissionData(this.SmallProducerSubmissionData);
 
             var model = await submissionService.Submissions(year);
+            using (var client = this.apiClient())
+            {
```

Comment:
@will-hope98 Do we need to do it for all of the tab actions in the ProducerController as I suspect if you switch tabs the button will vanish.  Maybe add a parameter to WithSubmissionData with the API year and set it in there, which would show the external places that it needs to be updated. Perhaps this could be a nullable int then it could be defaulted to null so any internal screens dont need a value passed.